### ISSUE TITLE
enhanced error messaging around scratch containers

### ISSUE THREAD

**Issue by mitchdraft** [solo-io/squash#56]
If a container was built from "scratch" some of the common utils that squash requires will not exist (such as `ls`)

During squash debugging, the user will need to replace that container with one built from a different distro, such as alpine

That message should replace the current error:
```
WARN[0000] getNSAlphav1 error                            err="mnt namespace not found"
FATA[0000] debug failed!                                 err="mnt namespace not found"
```

**Comment by TheDiveO** [comment]
I'm wondering why there is the need for an `ls` binary in the target container anyway? It's trying to do a `ls -l /proc/self/ns/` in order to get the netns inode # (identifier). And this can be perfectly done from outside the container from a privileged container: it's a readlink(2) operation which is needed here. And since the PID for attaching to the container is from the outer PID namespace anyway why not fetching the netns ID from /proc/$PID/ns anyway?

**Comment by mitchdraft** [comment]
good idea, worth a try, it'd be nice to remove all target container dependencies.
any thoughts @yuval-k
